src/components/ActionsMenu/Item/ActionsMenuItem.st.css

```diff
@@ -0,0 +1,77 @@
+:import {
+    -st-from: "../../../common/formatters.st";
+    -st-named: color, fallback, font, applyOpacity;
+}
+
+:vars {
+    /*Defaults*/
+    ItemFontDesktop: "{theme: 'Body-M', size: '16px', lineHeight: '24px'}";
+    ItemSubtitleFontDesktop: "{theme: 'Body-M', size: '14px', lineHeight: '20px'}";
+    ItemFontMobile: "{theme: 'Body-M', size: '14px', lineHeight: '24px'}";
+    ItemSubtitleFontMobile: "{theme: 'Body-M', size: '12px', lineHeight: '20px'}";
+}
+
+:vars {
+    /*Overrides*/
+    ItemFont:  --overridable;
+    ItemSubtitleFont:  --overridable;
+}
+
+.root {
+    -st-states: disabled, mobile;
+    height: 40px;
+    padding: 8px 24px;
+    display: flex;
+    cursor: pointer;
+    position: relative;
+}
+
+.root:hover {
+    background-color: rgba(0,0,0, 0.06);
+}
+
+.root .container {
+    display: flex;
+    flex: 1;
+    position: absolute;
+}
+
+.root:disabled {
+    opacity: 0.3;
+    cursor: default;
+}
+
+.root:disabled:hover {
+    background-color: inherit;
+}
+
+.root .text {
+    display: flex;
+    flex-direction: column;
+    color: color(color-5);
+    width: 100%;
+}
```

Comment:
Theoretically, a more "correct stylable" way to write this is by extending the `Text` component:
```css
:import {
  -st-from: '../../Text/Text.st.css';
  -st-default: Text;
}

.text {
  -st-extends: Text;
  -st-mixin: Text(
    MainTextColor color(color-5)
  );
  ...
}
```
(by the way, the `.root` isn't needed here, stylable knows to deal with it)

Reply:
ok, will use `Text`
but in that case `.root` is required because we get an error otherwise

Reply:
That's weird 🤔